packages/checkbox/test/checkbox.test.ts

```diff
@@ -28,6 +28,11 @@ function inputForCheckbox(checkbox: Checkbox): HTMLInputElement {
     return checkbox.shadowRoot.querySelector('#input') as HTMLInputElement;
 }
 
+function labelForCheckbox(checkbox: Checkbox): HTMLLabelElement {
```

Comment:
This approach expands on reaching into the Shadow DOM from a very early history on the library, would it be less invasive to use the accessibility snap shot were we can ensure the relationships between these elements, rather than just the presence of attributes? Checkout https://github.com/adobe/spectrum-web-components/blob/main/packages/radio/test/radio-group.test.ts#L301 for an example of this.